app/controllers/digital-wallet/post-disable-apple-pay-controller.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+// Local dependencies
+const paths = require('../../paths')
+const { ConnectorClient } = require('../../services/clients/connector_client')
+const auth = require('../../services/auth_service')
+const { CORRELATION_HEADER } = require('../../utils/correlation_header.js')
+const connector = new ConnectorClient(process.env.CONNECTOR_URL)
+
+module.exports = (req, res) => {
+  const gatewayAccountId = auth.getCurrentGatewayAccountId(req)
+  const correlationId = req.headers[CORRELATION_HEADER] || ''
+  const disableApplePayBoolean = connector.toggleApplePay(gatewayAccountId, false, correlationId)
+  disableApplePayBoolean.then(() => {
+    req.flash('generic', '<h2>Apple Pay successfully disabled.</h2>')
```

Comment:
`flash`ing markup seems like a bad precedent to set -- in general the controller should not be concerned with how anything is rendered. 

Could this be picked up and formatted by the template?

Reply:
I agree this is a bit irregular but we do it already _all over the place_ e.g.  
https://github.com/alphagov/pay-selfservice/blob/b78d02a609896bb1208d256428eefd4fa2f9d203/app/controllers/credentials_controller.js#L100
can you think of a better way to do this @jonheslop

Reply:
Yeah it’s bad, but maybe I can come up with something better in a separate PR after this is merged

something like 
`req.flash('genericError', {heading:"Please enter a valid username",body:""})`

Reply:
Would also mean we wouldn’t have to have `| safe` in the flash.njk template

Reply:
@jonheslop 👍 I would rather we raise a tech debt card for this and fix in a separate PR as this pattern is not restricted to this story.

Reply:
...OK I made a tech debt card now: https://trello.com/c/ro30NARU/50-selfservice-uses-in-line-html-in-controllers-to-render-error-messages

Reply:
👍